tgui/packages/tgui-panel/styles/components/Stat.scss

```diff
@@ -43,6 +43,16 @@ $border-color: color.scale(colors.fg(colors.$primary), $lightness: 75%) !default
     rgba(#e6a648, 0) 40%);
 }
 
+.StatBorder_br {
+  border-left: (1em / 6) solid #e6a648;
```

Comment:
Change this to 844cc3 too or something unique for be button, keep the bar the same colour as the background.